orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Please write the equations in a new line each.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>will do, and to clarify that new line does not mean new cell?</p>

Reply:
No need to make a new cell. Just a gap line like so:
some text

```{math}
```